repos/coreclr.proj

```diff
@@ -11,6 +11,8 @@
     <LatestCommit>36f70fa4be4bd37d4d76d06dd2cc433ff46351bd</LatestCommit>
     <VersionSeedDate>2017-07-19</VersionSeedDate>
     <OfficialBuildId>20170719-03</OfficialBuildId>
+    <!-- CoreCLR is part of ProdCon, but source-build fixes haven't been in a build yet. -->
```

Comment:
Will be fixed with https://github.com/dotnet/source-build/pull/384

Reply:
Doesn't look like it, but I'm rewording the comment because I think it was unclear: `ProdCon is behind source-build: temporarily disable auto-update.`

Reply:
Source-build currently points to the latest commit in coreclr release/2.1 branch. What is missing?

Reply:
Oh.. I think I understand your comment now. Your correct there hasn't been a published prodcon build with this sha yet. That should be true for corefx as well.

Reply:
I see you disabled corefx as well.

Reply:
Yep, you got it. Sorry for the confusion.